myria3d/pctl/datamodule/hdf5.py

```diff
@@ -34,13 +33,13 @@ def __init__(
         pre_filter: Optional[Callable[[Data], bool]] = pre_filter_below_n_points,
         tile_width: Number = 1000,
         subtile_width: Number = 50,
-        subtile_shape: SHAPE_TYPE = "square",
         subtile_overlap_train: Number = 0,
         subtile_overlap_predict: Number = 0,
         batch_size: int = 12,
         num_workers: int = 1,
         prefetch_factor: int = 2,
         transforms: Optional[Dict[str, TRANSFORMS_LIST]] = None,
+        **kwargs,
```

Comment:
This is to ensure backward compatibility, but was not tested with older configs.